tests/test_web_app_handler.py

```diff
@@ -54,6 +54,7 @@ def test_not_completed_workflow():
     assert http_code == 200
 
 
+@pytest.mark.skip(reason="Not so important")
```

Comment:
These tests are not the best since they're testing malformed webhook events instead of using JSON schema validation.